conf/messages.en

```diff
@@ -630,9 +630,9 @@ partTimeHours.heading.singleDay = What hours did this employee actually work on
 partTimeHours.partial = These are the dates from the pay period {0} to {1} that fall inside the claim period.
 partTimeHours.p1 = Hours
 partTimeHours.hint = Enter hours in decimal format
-partTimeHours.error.required = Enter the part time hours for this pay period
-partTimeHours.error.min = The part time hours cannot be negative
-partTimeHours.error.max = Enter a real part time hours
+partTimeHours.error.required = Enter the hours this employee actually worked in this pay period
```

Comment:
can you replace them for welsh as well ? 
just add the english and i will take care of welsh later